cmd/check.go

```diff
@@ -190,6 +195,34 @@ func getInitialRuntimeData(cmd *cobra.Command) shared.ImportLoopData {
 	}
 }
 
+func detach(cmd *cobra.Command) {
+	isDetached, err := cmd.Flags().GetBool("detached")
+	if err != nil {
+		panic(err)
+	}
+
+	if !isDetached {
```

Comment:
Could you please reduce the scope of this `if` statement to just:
``` go
		args = append(os.Args, "--detached")
```

Reply:
do you mean chaning it to:
```suggestion
	if isDetached {
	    return;
	 }
```